packages/template-ui/src/views/Home.vue

```diff
@@ -139,13 +139,18 @@ export default {
     },
     fetchContentNodes() {
       this.loadingContentNodes = true;
-      return window.kolibri.getContentByFilter({ parent: 'self', onlyContent: true })
+      const options = this.hasFlatGrid ? { onlyContent: true } : { parent: 'self', onlyContent: true }
+      return window.kolibri.getContentByFilter(options)
         .then((page) => {
           this.contentNodes = page.results;
           this.loadingContentNodes = false;
         });
     },
     fetchSectionNodes() {
+      if (this.hasFlatGrid) {
+        this.loadingSectionNodes = false;
+        return;
```

Comment:
This creates a lint error: https://eslint.org/docs/rules/consistent-return

This should return a promise to make it consistent with following calls, but it's only used in the `Promise.all` this could be fixed just with an explicit `return null`.

Reply:
OK! Fixed, and also fixed the returns of the method above `fetchCarouselNodes`, which was not returning the promises.

But again, this doesn't produce a lint error for me. Have you checked your eslint settings? Also I don't see the CI running for this PR, that would be the source of truth.